source/iface/group.rst

```diff
@@ -4,114 +4,316 @@
 
 .. _group:
 
-.. rst-class:: api-class
-
 ===============
 ``sycl::group``
 ===============
 
 ::
 
-   template <int dimensions = 1>
+   template <int Dimensions = 1>
    class group;
 
-.. rubric:: Template parameters
+The ``group`` encapsulates all functionality required to represent
```

Comment:
From my perspective, it's better to write ``sycl::group`` and the same with all other SYCL names, but it's still in the discussion, so if it's considered. we'll fix it anyway.
Also, my recommendation to write for ``id`` or ``range`` something like `` :ref:`id` `` and `` :ref:`range` ``, so that user could navigate to the needed chapter. But it's just a recommendation and if SYCL want us to implement it, we'll fix it fast.